plugins/PagePerformance/lang/en.json

```diff
@@ -0,0 +1,34 @@
+{
+    "PagePerformance": {
+        "ColumnAveragePageLoadTime": "Avg. page load time",
+        "ColumnAveragePageLoadTimeDocumentation": "Average time how long it takes from requesting a page until the page is fully rendered within the browser",
+        "ColumnAverageTimeLatency": "Avg. latency time",
+        "ColumnAverageTimeLatencyDocumentation": "Average time how long it takes the response to get to the user’s device. Includes the time it takes for the request to get to the server, it takes the server to render a response, and until the first byte of that response gets back to the user’s device.",
+        "ColumnAverageTimeTransfer": "Avg. transfer time",
+        "ColumnAverageTimeTransferDocumentation": "Average time how long it takes the browser to download the response from the server.",
+        "ColumnAverageTimeDomProcessing": "Avg. DOM processing time",
+        "ColumnAverageTimeDomProcessingDocumentation": "Average time how long the browser spends loading the webpage until the user can starting interacting with it.",
+        "ColumnAverageTimeDomCompletion": "Avg. DOM completion time",
+        "ColumnAverageTimeDomCompletionDocumentation": "Average time how long it takes for the browser to load media and execute any Javascript code listening for the DOMContentLoaded event.",
+        "ColumnAverageTimeOnLoad": "Avg. on load time",
+        "ColumnAverageTimeOnLoadDocumentation": "Average time how long it takes the browser to execute Javascript code waiting for the window.load event.",
```

Comment:
Same here. Be good to mention eg since the requesting the first byte or since the dom was loaded etc. BTW I suppose this could be also include loading images etc?

Reply:
No. The Onload time is only the time the browser spends to execute the javascript callbacks that were listening to the window.load event. If the javascript requests any stuff synchronous, that would be counted indirectly as well. But for most pages that shouldn't be the case.